Sources/LocalReceiptParsing/BasicTypes/AppleReceipt.swift

```diff
@@ -40,7 +40,7 @@ struct AppleReceipt: Equatable {
 
     let bundleId: String
     let applicationVersion: String
-    let originalApplicationVersion: String
+    let originalApplicationVersion: String?
```

Comment:
See PR description, `originalApplicationVersion` is apparently not reported in SKTest receipts

Reply:
yeah exploring enforcing with the compiler is a good idea. I'm going to add that to the list of things to do in the follow-ups